test-manager/src/tests/account.rs

```diff
@@ -11,22 +12,34 @@ pub async fn test_login(
     _rpc: ServiceClient,
     mut mullvad_client: ManagementServiceClient,
 ) -> Result<(), Error> {
+    const THROTTLE_RETRY_DELAY: Duration = Duration::from_secs(120);
+
     //
     // Instruct daemon to log in
     //
 
-    // TODO: Test too many devices, removal, etc.
```

Comment:
Do we not want to do these TODOs?

Reply:
We already test removal elsewhere, and we track the other stuff in better places.

Reply:
Alright